package/network/utils/iptables/patches/101-remove-check-already.patch

```diff
@@ -1,9 +1,11 @@
---- a/libxtables/xtables.c
-+++ b/libxtables/xtables.c
-@@ -903,12 +903,6 @@ static void xtables_check_options(const
+Index: iptables-1.8.6/libxtables/xtables.c
```

Comment:
No. run make package/iptables/refresh

Reply:
Ah you did. Nvm.

Reply:
Ya I had to alter, wouldnt auto refresh :(